packages/webdriverjs/src/index.ts

```diff
@@ -291,4 +291,10 @@ export default class AxeBuilder {
   }
 }
 
+// ensure backwards compatibility with commonJs default export
+if (typeof module === 'object') {
+  module.exports = AxeBuilder;
+  module.exports.AxeBuilder = AxeBuilder;
```

Comment:
If someone got broken by our earlier accidental breaking change and worked around it by switching to `require('@axe-core/webdriverjs').default`, this update will break them again. I think we should avoid that.

If we instead do this...

```suggestion
  module.exports = AxeBuilder
  module.exports.default = AxeBuilder;
  module.exports.AxeBuilder = AxeBuilder;
```

...I think it allows us to support all 3 of `require()`, `require().default`, and `{ AxeBuilder } = require()` simultaneously.

I strongly think we should do that for webdriverjs, but even in all the other packages I think it would be a reasonable choice to make `require('packagename')` ergonomic.

This is the modified commonjsTest file I used to validate this approach:

```js
// ensure backwards compatibility of commonJs format
const implicitDefaultExport = require('../dist/index.js');
const explicitDefaultExport = require('../dist/index.js').default;
const { AxeBuilder } = require('../dist/index.js');
const assert = require('assert');

assert(typeof AxeBuilder === 'function', 'named export is not a function');

assert(typeof implicitDefaultExport === 'function', 'implicit default export is not a function');
assert(
  implicitDefaultExport === AxeBuilder,
  'implicit default and named export are not the same'
);

assert(typeof explicitDefaultExport === 'function', 'explicit default export is not a function');
assert(
  explicitDefaultExport === AxeBuilder,
  'explicit default and named export are not the same'
);
```